SharedLibs/TileMapHelper/src/render/TileMapManager.ts

```diff
@@ -177,4 +177,9 @@ export class TileMapManager {
       callback
     );
   }
+
+  clearCaches(): void {
+    this._tileMapCache = new ResourceCache<EditableTileMap>();
```

Comment:
does anything need to be done to the previous ResourceCaches ? We assume they will be garbage collected I guess?

Reply:
I thought of this but yes I think it will be collected